apps/cowswap-frontend/src/modules/tradeWidgetAddons/containers/SettingsTab/index.tsx

```diff
@@ -97,7 +97,7 @@ function SettingsTabController({ buttonRef, children }: SettingsTabControllerPro
 
   const toggleMenu = useCallback(() => {
     buttonRef.current?.dispatchEvent(new Event('mousedown', { bubbles: true }))
-  }, [buttonRef.current])
+  }, [buttonRef])
```

Comment:
Can't you use current here?

Reply:
buttonRef is a component prop here, so we have to add it in the deps array